### PR TITLE
Feature/build image for scone enclave

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+```
+
+## TEE (protected) mode
+To convert the application into **TEE** mode, first, it needs to be
+build in **Standard** mode as instructed in the section above.
+Then the produced image is converted using `sconify.sh` script into
+a newly created TEE enabled image `tee-go-hello-world`:
+
+### Build (conversion)
+The script can edited to change parameters like **heap size**, new
```
Comment (mcornaton) [inline README.md:41]:
```suggestion
The script can be edited to change parameters like **heap size**, new
```

**Review by mcornaton — APPROVED** [review]

File: scone/sconify.args [hunk 2]
```diff
@@ -0,0 +1,13 @@
+--name=go-hello-world
+--from=${IMG_FROM}
+--to=${IMG_TO}
+--binary-fs
```
Comment (jeremyjams) [inline scone/sconify.args:4]: Binary FS should not be used for compiled applications (natively executable on host):
"This is a particularly appealing solution for interpreted languages such as Python or Java where the actual program code (e.g. PyTorch or Kafka) is not part of the interpreter binary but read from byte code files in the file system. " (https://sconedocs.github.io/binary_fs/)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,69 @@
...
+The application can be tested locally to make sure it is well setup:
+```
+rm -rf /tmp/iexec_out && \
+docker run \
+        --rm \
+        -e IEXEC_OUT=/iexec_out \
+        -e IEXEC_IN=/iexec_in \
+        -v /tmp/iexec_out:/iexec_out \
+        -v $(pwd)/iexec_in:/iexec_in \
+        --device /dev/isgx \
```
[the remark this answers is not in the document]
Comment (jbern0rd) [reply README.md]: Replaced with native devices in 9227e40

File: scone/sconify.args (hunk 2 above)
Comment (jbern0rd) [reply scone/sconify.args]: Removed in 2736fc5

File: README.md (hunk 1 above)
Comment (jbern0rd) [reply README.md]: Thanks

**Review by james-toussaint — APPROVED** [review]
Thank you :+1:

**Pull request merged**